wiki/Getting-Started-SwiftUI.md

```diff
@@ -0,0 +1,163 @@
+
+## IMPORTANT NOTE
+
+SwiftCurrent is so convenient that you may miss the couple lines that are calls to the library.  To make it easier, we've marked our code snippets with `// SwiftCurrent` to highlight items that are coming from the library.
+
+## Create your views
+
+Create two views that implement [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html).
+
+```swift
+import SwiftUI
+import SwiftCurrent
+
+struct FirstView: View, FlowRepresentable { // SwiftCurrent
+    typealias WorkflowOutput = String // SwiftCurrent
+    weak var _workflowPointer: AnyFlowRepresentable? // SwiftCurrent
+
+    @State private var email = ""
+    private let name: String
+    
+    init(with name: String) { // SwiftCurrent
+        self.name = name
+    }
+
+    var body: some View {
+        VStack {
+            Text("Welcome \(name)!")
+            TextField("Enter email...", text: $email)
+                .textContentType(.emailAddress)
+            Button("Save") { proceedInWorkflow(email) }
+        }
+    }
+}
+
+struct FirstView_Previews: PreviewProvider {
+    static var previews: some View {
+        FirstView(with: "Example Name")
+    }
+}
+
+struct SecondView: View, FlowRepresentable { // SwiftCurrent
+    typealias WorkflowOutput = String // SwiftCurrent
+    weak var _workflowPointer: AnyFlowRepresentable? // SwiftCurrent
+
+    private let email: String
+
+    init(with email: String) { // SwiftCurrent
+        self.email = email
+    }
+
+    var body: some View {
+        VStack {
+            Button("Finish") { proceedInWorkflow(email) }
+        }
+    }
+
+    func shouldLoad() -> Bool { // SwiftCurrent
+        email.lowercased().contains("@wwt.com")
+    }
+}
+
+struct SecondView_Previews: PreviewProvider {
+    static var previews: some View {
+        SecondView(with: "Example.Name@wwt.com")
+    }
+}
+```
+
+### Let's talk about what is going on with these views
+
+#### **Why is `_workflowPointer` weak?**
+
+<details>
+
+The [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) protocol requires there to be a `_workflowPointer` on you object, but protocols cannot enforce you to use `weak`. If you do not put `weak var _workflowPointer`, the [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) will end up with a strong circular reference.
+</details>
+
+#### **What's this `shouldLoad()`?**
+
+<details>
+
+It is part of the [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) protocol. It has default implementations created for your convenience but is still implementable if you want to control when a [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) should load in the work flow.  It is called after `init` but before `body` in SwiftUI.
```

Comment:
```suggestion
It is part of the [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) protocol. It has default implementations created for your convenience but is still implementable if you want to control when a [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) should load in the Workflow.  It is called after `init` but before `body` in SwiftUI.
```

Reply:
It looks like your change is implying that we should be referencing "Workflow" the type and not "work flow" the concept?

Reply:
```suggestion
It is part of the [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) protocol. It has default implementations created for your convenience but is still implementable if you want to control when a [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) should load in the workflow.  It is called after `init` but before `body` in SwiftUI.
```

Seems like concept of workflow is what was meant here.

Reply:
Yes +1 to @wiemerm's suggestion